src/main/config/config.c

```diff
@@ -832,6 +832,10 @@ void activateConfig(void)
 
 void validateAndFixConfig(void)
 {
+    if (config->motor_pwm_protocol = PWM_TYPE_BRUSHED && escAndServoConfig->mincommand < 1000) {
```

Comment:
`if (config->motor_pwm_protocol == PWM_TYPE_BRUSHED` ...
not
`if (config->motor_pwm_protocol = PWM_TYPE_BRUSHED` ...